BuildAndServe.bat

```diff
@@ -0,0 +1,3 @@
+.\docfx\docfx.exe docfx.json --serve
```

Comment:
We suggest to add docfx.exe to environment path, then use docfx docfx.json --serve directly? Not all users would place latest release to docfx folder, so this might not be normal case...

---
In reply to: [71216491](https://github.com/docascode/docfx-seed/pull/9#discussion_r71216491) [](ancestors = 71216491)